Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris.yml

```diff
@@ -37,16 +37,18 @@ configuration:
   required: false
   type: 0
 
-defaultclassifier: Rubrik Polaris Radar - Classification
-defaultmapperin: Rubrik Polaris Radar - Mapping
+defaultclassifier: c3e38fe1-e429-4836-8050-a8d399518fe8
+defaultmapperin: 97b3f7dc-5bdb-40c8-819b-8c1beb95cb09
```

Comment:
@ShahafBenYakir correct me if I'm wrong, the best practice here would be to change the IDs of the classifier/mapper and not vice-versa... but will it be OK if customers already depend on these IDs?
If this is not too late to do it, we can set a call to clarify what's needed to be done here